doc/source/whatsnew/v0.18.1.txt

```diff
@@ -76,8 +76,7 @@ API changes
 
 
 
-
-
+- ``read_csv`` now raises an ``EmptyDataError`` in response to empty columns or header (:issue:`12506`)
```

Comment:
I added examples to the whatsnew.  However, as I've never done it before, not sure if I did it entirely correctly.